pandas/core/arrays/arrow/array.py

```diff
@@ -48,16 +53,130 @@
 
 class ArrowExtensionArray(ExtensionArray):
     """
-    Base class for ExtensionArray backed by Arrow array.
+    Base class for ExtensionArray backed by Arrow ChunkedArray.
     """
 
     _data: pa.ChunkedArray
 
-    def __init__(self, values: pa.ChunkedArray) -> None:
-        self._data = values
+    def __init__(self, values: pa.Array | pa.ChunkedArray) -> None:
+        if pa_version_under1p01:
+            msg = "pyarrow>=1.0.0 is required for PyArrow backed ArrowExtensionArray."
+            raise ImportError(msg)
+        if isinstance(values, pa.Array):
+            self._data = pa.chunked_array([values])
+        elif isinstance(values, pa.ChunkedArray):
+            self._data = values
+        else:
+            raise ValueError(
+                f"Unsupported type '{type(values)}' for ArrowExtensionArray"
+            )
+        self._dtype = ArrowDtype(self._data.type)
+
+    @classmethod
+    def _from_sequence(cls, scalars, *, dtype: Dtype | None = None, copy=False):
```

Comment:
@mroeschke i just tried the following and got an ArrowInvalid exception

```
arr = pa.array([1, 2, 3])
ea = pd.core.arrays.ArrowExtensionArray._from_sequence(arr)
```

should this work?

update: looks like just `__init__` works fine here.  still surprising that from_sequence doesnt

Reply:
Related to https://github.com/pandas-dev/pandas/issues/48238, I hadn't really anticipated users passing pyarrow arrays but I suppose this should be supported.